R/deploymentTarget.R

```diff
@@ -72,6 +70,29 @@ deploymentTarget <- function(recordPath = ".",
   }
 }
 
+deploymentTargetForApp <- function(appId,
+                                   appTitle = NULL,
+                                   account = NULL,
+                                   server = NULL) {
+  accountDetails <- accountInfo(account, server)
+  client <- clientForAccount(accountDetails)
+
+  withCallingHandlers(
```

Comment:
This looks really similar to the (now uncalled? IDE-only?) `getAppById`.

Reply:
Looks like it's still use: https://github.com/rstudio/rstudio/blob/ee56d49b0fca5f3d7c3f5214a4010355d1bb0212/src/cpp/session/modules/SessionRSConnect.R#L213-L214. But should move it to `ide.R`

Reply:
I've made a note to rework `getAppById()`